### ISSUE TITLE
URLs containing only fragments should not be rendered

### ISSUE THREAD

**Issue by twpayne** [charmbracelet/glamour#43]
The input Markdown:

```
<!--- toc --->
* [Getting started](#getting-started)
* [Developing locally](#developing-locally)
* [Documentation changes](#documentation-changes)
* [Contributing changes](#contributing-changes)
* [Managing releases](#managing-releases)
* [Packaging](#packaging)
* [Updating the website](#updating-the-website)
```

with the default `dark` style is rendered as:

<img width="371" alt="glamour-fragments" src="https://user-images.githubusercontent.com/6942/71525097-37070a80-28c8-11ea-9956-026f6724423b.png">

Since the URLs contain only document-relative fragments, they are not useful to click on or be copied to be opened elsewhere, and so should not be rendered.

**Comment by meowgorithm** [comment]
Totally. Looks like the rendering has changed, but this is dup of #4.

**Comment by muesli** [comment]
@meowgorithm Technically it's still the same rendering, I just assume in @twpayne's case there wasn't a BaseURL set on the renderer.

It is however not quite the same issue: while the anchored URLs in #4 could still be useful to other users of the renderer - you could open them in a browser after all - pure anchors as shown in the screenshot above are indeed entirely pointless.

**Comment by muesli** [comment]
Fixed by 4c42a50.
